pages/task/read/content/project.js

```diff
@@ -10,7 +10,9 @@ module.exports = {
   continuation: {
     title: 'This is a project continuation',
     licence: 'From the licence',
-    expiry: 'Expiry'
+    expiry: 'Expiry',
+    rte: 'Original input',
+    label: 'Please state the licence number and expiry date of all these licences.'
```

Comment:
Thanks for not trying to make this clever.

Reply:
it would have to have been clever as this question doesn't exist anymore